src/content/ccip/tutorials/send-arbitrary-data-receipt-acknowledgment.mdx

```diff
@@ -0,0 +1,466 @@
+
+```solidity
+// Emitted when an acknowledgment message is successfully sent back to the sender contract.
+// This event signifies that the Acknowledger contract has recognized the receipt of an initial message
+// and has informed the original sender contract by sending an acknowledgment message,
+// including the original message ID.
+event AcknowledgmentSent(
+  bytes32 indexed messageId, // The unique ID of the CCIP message.
+  uint64 indexed destinationChainSelector, // The chain selector of the destination chain.
+  address indexed receiver, // The address of the receiver on the destination chain.
+  bytes32 data, // The data being sent back, usually containing the message ID of the original message to acknowledge its receipt.
+  address feeToken, // The token address used to pay CCIP fees for sending the acknowledgment.
+  uint256 fees // The fees paid for sending the acknowledgment message via CCIP.
+);
+```
+
+1. Copy your _acknowledger_ contract address from Remix. Open the [_Polygon Mumbai_ explorer](https://mumbai.polygonscan.com) and search for your deployed _acknowledger_ contract, then click on the **Events** tab.
+
+   <ClickToZoom
+     src="/images/ccip/tutorials/polygon-mumbai-explorer-acknowledger-events.webp"
+     alt="Chainlink CCIP - Acknowledger Get Last Received Message Details"
+     style="max-width: 70%;"
+   />
+
+   The first indexed topic (`topic1`) in the `AcknowledgmentSent` event is the acknowledgment message ID sent to the _message tracker_ contract on _Ethereum Sepolia_. In this example, the message ID is `0x0defcd4d21415378e7136b7300940fb8c0e9e046002aa9b54a0362ecb5cee0ec`.
+
+1. Copy your own message ID from the indexed `topic1` and search for it in the [CCIP explorer](https://ccip.chain.link/).
+
+   <ClickToZoom
+     src="/images/ccip/tutorials/ccip-explorer-mumbai-sepolia-success.webp"
+     alt="Chainlink CCIP - CCIP Explorer Mumbai to Sepolia Transaction Success"
+     style="max-width: 70%;"
+   />
+
+   When the transaction is marked with a "Success" status on the CCIP explorer, the CCIP transaction and the destination transaction are complete. The _message tracker_ contract has received the message from the _acknowledger_ contract.
+
+#### Final status check
+
+Upon reception of the acknowledgment message, the _message tracker_ `ccipReceive` function updates the initial message status to `2`, which corresponds to the `Confirmed` status as defined by the `MessageStatus` `enum` and emits a `MessageConfirmed` event.
+
+1. On the **Deploy & Run Transactions** tab in Remix, expand your _message tracker_ contract in the **Deployed Contracts** section.
+
+1. Paste the message ID you copied from the CCIP explorer as the argument in the **messagesInfo** getter function. Click **messagesInfo** to read the message status. It returns `status 2` and the acknowledger message ID that confirms this status.
```

Comment:
Make sure users understand that we are talking here about the first CCIP message ID (so that you don't have users who mistakenly copy the messageID of the message that was sent by the Acknowledger contract)